english/eating-walls/find-a-toilet/door/run-in/push-lever/bear.md

```diff
@@ -0,0 +1 @@
+You walk over to the wall and push the lever. You hear a creak as the door behind you opens, and a growl as an angry polar bear comes charging at you, grabs you and diligently goes about the process of tearing you limb from limb. That was your last breath you realize as the world slowly blacks out.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.